src/server_manager/ui_components/app-root.html

```diff
@@ -59,126 +72,178 @@
         height: 20px;
         width: 20px;
       }
-      #settingsDrawer {
-        z-index: 1000;
-        margin-top: 12px;
+      #appDrawer {
+        --app-drawer-content-container: {
+          background-color: var(--background-contrast-color);
+        };
       }
-      #settingsDrawer paper-listbox {
-        color: var(--light-gray);
+      #appDrawer paper-listbox {
+        color: var(--medium-gray);
+        background-color: var(--background-contrast-color);
       }
-      #settingsDrawer paper-icon-item {
+      #appDrawer paper-icon-item {
         display: block;
         cursor: pointer;
         padding: 12px 24px 12px 24px;
         font-size: 14px;
       }
-      #settingsDrawer iron-icon {
+      #appDrawer iron-icon {
         margin-right: 12px;
       }
-      #settingsDrawer paper-icon-item:hover,
-      #settingsDrawer paper-icon-item iron-icon:hover {
+      #appDrawer paper-icon-item:hover,
+      #appDrawer paper-icon-item iron-icon:hover {
         color: var(--primary-green);
       }
-      #settingsDrawer a {
+      #appDrawer a {
         color: inherit;
       }
-      #settingsDrawer paper-icon-item:focus,
-      #settingsDrawer a:focus {
+      #appDrawer paper-icon-item:focus,
+      #appDrawer a:focus {
         outline: none;
       }
-      #settingsDrawer #signedIn {
+      #appDrawer #signedIn {
         text-align: center;
       }
-      #settingsDrawer #signedIn img {
+      #appDrawer #signedIn img {
         height: 65px;
         margin-top: 12px;
       }
-      #settingsDrawer #signOutButton  {
+      #appDrawer #signOutButton  {
         font-weight: bold;
         margin-bottom: 24px;
         cursor: pointer;
         text-transform: uppercase;
       }
-      #settingsDrawer #signOutButton:hover {
+      #appDrawer #signOutButton:hover {
         color: var(--primary-green);
       }
-      app-toolbar paper-icon-button {
-        /* needed so the "Outline Manager" title appears centered in the window. */
-        position: absolute;
-      }
-      app-toolbar [main-title] {
-        text-align: center;
+      app-drawer-layout app-toolbar {
+        background-color: var(--background-contrast-color);
+        border-bottom: 1px solid rgba(200, 200, 200, 0.12);
       }
       app-toolbar [main-title] img {
         height: 16px;
         margin-top: 8px;
       }
+      .side-bar-margin {
+        margin-left: var(--status-bar-width);
+      }
+      #sideBar {
+        --app-drawer-width: var(--status-bar-width);
```

Comment:
This variable doesn't seem to be used anywhere else?

Reply:
Polymer uses to set the width of the drawer.